mk/spksrc.service.mk

```diff
@@ -205,7 +205,7 @@ endif
 # - firewall rules/port definitions (DSM >= 6.0-5936)
 # - data share worker (DSM 7, optional for DSM 6)
 # - usr local links (DSM >= 6.0-5941)
-# - certificate config
+# - certificate config (DSM < 7, available only for Synology packages since DSM 7)
```

Comment:
```
# - certificate config (DSM < 7, available only for Synology packages before DSM 7)
```

Reply:
That's not correct though.  We support it only for DSM 6, since the function is not available for community packages on DSM 7.

Reply:
I know, I know. It's more a semantic mismatch, as "since DSM 7" means "DSM >= 7".

Reply:
I think you're just missing the emphasis here.   It's meant to be:

> available **only** for **Synology** packages since DSM 7

and NOT

> available *only* for Synology **packages since DSM 7**

How could that be made clearer?

Reply:
@hgy59 Re-phrased in latest commit, hope that's clearer now.